### ISSUE TITLE
Add an action to recheck a translation for warnings

### ISSUE THREAD

**Issue by pedro-mendonca** [GlotPress/GlotPress-WP#1556]
Many project have very old strings, like WP core on wp.org, that probably had warnings that were somehow ignored, or have errors that exist before the matching warnings were developed and deployed.

Let's say we add a [check for missing dots](https://github.com/GlotPress/GlotPress/issues/1555), it will test only the new translations, the approved ones won't ever be tested against this new check.

Some of these errors are easily found if one reimports the project, but there is no way to retest without reimporting.

Would be great to have a bulk action to retest all the strings of a translation, or a project, resetting all the ignored warnings so the old errors are made visible.

**Comment by NekoJonez** [comment]
Yeah, also... Let's try and avoid this recheck spamming Slack when this is implemented on wp.org. 

Also, I wouldn't set these fuzzy since if there is a good translation... It'd set it fuzzy. I'd throw it in changes requested with a tag of "automated test hit" or something like that.

**Comment by pedro-mendonca** [comment]
The criteria should be the same as when the translation is made/approved. It's just flagged with warning, the translation status is kept the same.

**Comment by amieiro** [comment]
It would be interesting to have a CLI command that we can execute with the cron, so we could check some projects once a week, a month, ... But if we test some projects once a week, and we always find some false positives, it will be an extra load of work for the validators (GTE/PTE at translate.w.org) each week to review this false warnings.

**Comment by pedro-mendonca** [comment]
If the validation rules have an ID, and a string has already a warning that was dismissed, doesn't make sense to be marked as having the same warning again. Only if another warning with different ID flags that string.
I don't know how the warnings are managed in the database, but if we can use an unique ID, seems doable.
